### ISSUE TITLE
VRTK ControllerTooltips positioning

### ISSUE THREAD

**Issue by beccannlittle** [Innoactive/VRTK-Windows-MR-Extension#7]
The [ControllerTooltips](https://github.com/thestonefox/VRTK/blob/master/Assets/VRTK/Documentation/API.md#controller-tooltips-vrtk_controllertooltips) prefab within VRTK by default look like this when used with WMR:

![front](https://user-images.githubusercontent.com/6414050/39481095-2005b286-4d1f-11e8-8f8a-c4b4dad531c8.png)
![back](https://user-images.githubusercontent.com/6414050/39481098-216a961e-4d1f-11e8-96e1-9104859efb92.png)

Does anyone have a solution for proper placement of the lines coming off the buttons for ControllerTooltips?

**Comment by tomwim** [comment]
You are using the Samsung Odyssey, I guess? And just to make sure because there are a lot of people who are confused with the whole WinMR VRTK stuff: You are using **VRTK** for **UWP** with this Extension (so **native Windows Mixed Reality**)?

So the problem with the VRTK Tooltips is that they get the exact position of the controller elements created by GLTF at runtime. This makes it quite inaccurate. You can look into the ControllerModel object. If you move around for example the "SELECT" child you'll see how the tooltip moves.

![ttipps](https://user-images.githubusercontent.com/3699271/39523085-b7f7520c-4e14-11e8-8b23-ba2767af0e95.png)
![ttipps_front](https://user-images.githubusercontent.com/3699271/39523112-c6b9e458-4e14-11e8-8ef7-00a0dfebf045.png)

BUT: looking at the screenshots you attached it seems that all tooltips go to the same "attach point". 
I assume in the VRTK_ControllerTooltips ```string elementPath = VRTK_SDK_Bridge.GetControllerElementPath(findElement, controllerHand, true);``` the elementPath is null. Maybe debug here and see if the ControllerElementPath can be found and to which point it's set.

To get a better result I did a workaround once for SteamVR Windows MR where I extended the VRTK_ControllerTooltip to set a (hard coded) offset to the controller elements. That worked quite well but needed some tweaking.

Thanks for the feedback and if you have more questions, feel free to ask! 👍

**Comment by beccannlittle** [comment]
Yes, Samsung Odyssey + native WMR (no SteamVR). We did try this with Odyssey + SteamVR and seemed to get more accurate tooltip placements.

We decided to take a pause on making the tooltips look nice, but when we come back to this I'll work on debugging where you suggested. If I find anything out I'll post it here.
